tests/mocks/TestObject.php

```diff
@@ -9,7 +9,8 @@
 class TestObject extends DataObject implements TestOnly
 {
     private static $db = [
-        'Title' => 'Varchar(255)'
+        'Title' => 'Varchar(255)',
```

Comment:
YASH! ShowInSearch!

Reply:
Yeah, will re-use this property to index only when its true in future works